src/main/java/com/github/shyiko/mysql/binlog/event/deserialization/AbstractRowsEventDataDeserializer.java

```diff
@@ -478,6 +489,65 @@ private Serializable deserializeTimeInteger(int value) {
         return timestamp != null ? new java.sql.Timestamp(timestamp) : null;
     }
 
+    /**
+     *  Binlog represents TIME_V1 as a LE 3-byte (24-bit) integer
```

Comment:
Is it really LE? I thought it was big endian since the hour since the hour is most significant and comes first